swsnu/swpp17-team5
# Google signin

closes #4 , #6 

- Use DRF
  - > Two scoops of Django : since 2013, 90~95% of new Django projects with an API use DRF.
- Add custom exceptions
- Add pytest, pytest-cov, factory boy for better testing
- Add error handling in chrome extension
- Use oauth token once for authentication with the server & revoke it instantly
- Can run tests locally by executing run_test.sh
- Should set these keys in .env file (in settings directory)
  - SECRET_KEY, DB_NAME, DB_USERNAME, DB_PASSWORD, DEBUG(should be true for now)
